inference.py

```diff
@@ -48,10 +49,43 @@
 
 parser.add_argument('--nosmooth', default=False, action='store_true',
 					help='Prevent smoothing face detections over a short temporal window')
+parser.add_argument('--convert_25', default=False, action='store_true',
+					help='Convert the input video to 25 fps')
 
 args = parser.parse_args()
 args.img_size = 96
 
+def check_fps(filename):
+	"""
+	Checks the fps of a video.
+	"""
+	cmd = f'ffprobe -v error -select_streams v -of default=noprint_wrappers=1:nokey=1 -show_entries stream=r_frame_rate {filename}'
+	try:
+		op = subprocess.check_output(cmd, shell = True).decode('utf-8')
+	except CalledProcessError as e:
+		print(e.output)
+	op = op.strip().split('/')
+	op = list(map(int, op))
+	return round(op[0] / op[1]) if op[1] != 0 else 30 # if the denominator is 0, don't throw an error, convert the vid to 25 fps.
```

Comment:
Could you please explain what this line does? Thanks!

Reply:
The output is returned as a string. For example '3000/101\n'. so I strip the '\n' and then split on '/'. This returns ['3000', '101']. Then I convert both of them to int and finally return the division. In some cases though, it cannot identify the fps and returns 0 / 0. In that case I return 30 (any number other than 25) so that the convert_to_25_fps function will convert it to 25. If the video is already 25 fps it won't make a difference.